scripts/coverage-report.template

```diff
@@ -17,6 +17,7 @@
 <body>
   <main>
     <h1>Coverage and Quality Reports</h1>
+    <p>Page last updated: </p>
```

Comment:
As shown on the landed commit, adding a stamp will continually fail the diff, except for things landed on the same day as the last time this was updated